SAM.Picker/GamePicker.cs

```diff
@@ -157,15 +154,10 @@ private void RefreshGames()
                     continue;
                 }
                 this._FilteredGames.Add(info);
+                this.AddGameToLogoQueue(info);
             }
 
-            this._GameListView.BeginUpdate();
             this._GameListView.VirtualListSize = this._FilteredGames.Count;
-            if (this._FilteredGames.Count > 0)
-            {
-                this._GameListView.RedrawItems(0, this._FilteredGames.Count - 1, true);
```

Comment:
This was one of the major UI freezes, and I'm not entirely sure what's the point of redrawing here is. Removing it doesn't produce any bugs that I could find.